### DISCUSSION TITLE
Any way to work with jpeg certificate templates?

### DISCUSSION THREAD

**Discussion by pranavirabatti** [Sunbird-Knowlg/Community#43] [General Discussions]
Currently, the system accepts the SVG format for certificate templates. Is there any way to work with jpeg files? Or any open source editor that will do this job of converting JPEG to SVG before uploading the template? 

@vinukumar-vs @AmiableAnil

**Comment by vinukumar-vs** [comment]
@pranavirabatti Only using SVG image certificate templates, we can modify the Cert title, Cert Course name, User Name etc.. 

We can't achieve this using any other image formats. Hence we support only SVG for certificate templates.

Help us to know what is your requirement/challenge. So that we can guide a better solution. 

@vrayulu, @maheshkumargangula, @AmiableAnil

**Reply by pranavirabatti** [reply]
We are trying to provide the functionality of uploading certificate templates from the admin dashboard. And the requirement is it should accept both of these file formats - SVG and jpeg. As you said only SVG is supported I was thinking to do the conversion with some external lib. Even this looks a bit daunting to me. Do you have any idea?

**Reply by vinukumar-vs** [reply]
Not sure jpeg to SVG image conversion is possible(may be). Assume even if you find the library to convert JPEG to SVG, how can you add placeholders to the converted SVG image? So that you can add specific text from the admin panel/API while generating certificate.

JPEG is not meant for templates.

**Reply by pallakartheekreddy** [reply]
@pranavirabatti As we haven't received a response from you, we will proceed to close this thread. If you have any further questions or require assistance in the future, please don't hesitate to start a new thread. Thank you.
